src/main/java/eu/europeana/metis/sandbox/service/dataset/DatasetReportServiceImpl.java

```diff
@@ -85,7 +95,18 @@ public DatasetInfoDto getReport(String datasetId) {
     recordsProcessedByStep.forEach((step, statusMap) -> addStepInfo(stepsInfo, statusMap, step,
         recordErrorsByStep));
 
-    return new DatasetInfoDto(dataset.getRecordsQuantity(), completedRecords, stepsInfo);
+    return new DatasetInfoDto(getPortalUrl(dataset, completedRecords),
+        dataset.getRecordsQuantity(), completedRecords,
+        stepsInfo);
+  }
+
+  private String getPortalUrl(DatasetEntity dataset, long completedRecords) {
+    long recordsQty = dataset.getRecordsQuantity();
+    if (recordsQty != completedRecords) {
+      return PORTAL_MSG;
+    }
+    var datasetId = dataset.getDatasetId() + SEPARATOR + dataset.getDatasetName() + SUFFIX;
+    return portalUrl + URLEncoder.encode(datasetId, StandardCharsets.UTF_8);
```

Comment:
This generates something like
`http://preview-metis/portal/search?view=grid&q=edm_datasetName:12_test*`
But the portal by default filter out content tier 0 results, should we be including them? I know what to add to the url but just want to confirm if it makes sense